django/contrib/admin/filters.py

```diff
@@ -153,26 +153,28 @@ def register(cls, test, list_filter_class, take_priority=False):
             cls._field_list_filters.append((test, list_filter_class))
 
     @classmethod
-    def create(cls, field, request, params, model, model_admin, field_path):
+    def create(cls, field, request, params, model, model_admin, field_path, title=None):
         for test, list_filter_class in cls._field_list_filters:
             if test(field):
-                return list_filter_class(field, request, params, model, model_admin, field_path=field_path)
+                return list_filter_class(
+                    field, request, params, model, model_admin, field_path=field_path, title=title
+                )
```

Comment:
This should just fit on one line I think. (It's exactly 120 characters. 🙂)

Reply:
Flake8 check is for 119 characters, so it fails the check at 120.